### PR TITLE
[9.0][ADD] Added BR Deliverable Default business_requirement_deliverable_default module for v9.

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
## WIP until #25  and #26  is merged

Migrated business_requirement_deliverable_default module from v8 to v9.
@dreispt @pedrobaeza @jbeficent @moylop260 Please review.

### CODE REVIEW

**Comment by elicoidal** [comment]
Let's wait for Travis and runbot to finish test but we could review and merge this one soon

File: business_requirement_deliverable_default/README.rst
Comment (elicoidal) [inline business_requirement_deliverable_default/README.rst]: I will improve the README

File: business_requirement_deliverable_default/__openerp__.py
Comment (elicoidal) [inline business_requirement_deliverable_default/__openerp__.py]: "Business Requirements Default Resource Lines in Deliverable Products"

**Review by elicoidal — APPROVED** [review]
I will improve the README. Other than that LGTM

**Comment by elicoidal** [comment]
@dreispt @pedrobaeza

**Review by dreispt — APPROVED** [review]
LGTM

**Comment by sudhir-serpentcs** [comment]
Rebased the PR.
@dreispt @pedrobaeza @jbeficent @moylop260 @elicoidal

File: business_requirement_deliverable_default/models/business_requirement_deliverable_default.py
Comment (astirpe) [inline business_requirement_deliverable_default/models/business_requirement_deliverable_default.py]: As per the [file naming](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming) guideline, could you split different models in separate files?

File: business_requirement_deliverable_default/views/business_requirement_deliverable_default.xml
Comment (astirpe) [inline business_requirement_deliverable_default/views/business_requirement_deliverable_default.xml]: As per the [file naming](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming) guideline of OCA, since the model name is `product.template`, the filename should be `product_template.xml`

**Review by astirpe — CHANGES_REQUESTED** [review]

File: business_requirement_deliverable_default/models/business_requirement_deliverable_default.py
Comment (elicoidal) [reply business_requirement_deliverable_default/models/business_requirement_deliverable_default.py]: @sudhir-serpentcs Can you attend @astirpe remarks so that we can merge this PR?

File: business_requirement_deliverable_default/views/business_requirement_deliverable_default.xml
Comment (JayVora-SerpentCS) [inline business_requirement_deliverable_default/views/business_requirement_deliverable_default.xml]: @astirpe Done

File: business_requirement_deliverable_default/models/business_requirement_deliverable_default.py
Comment (sudhir-serpentcs) [reply business_requirement_deliverable_default/models/business_requirement_deliverable_default.py]: @astirpe @elicoidal Done

Comment (elicoidal) [reply business_requirement_deliverable_default/models/business_requirement_deliverable_default.py]: @sudhir-serpentcs there might be a misunderstanding because I still all models in one file

Comment (sudhir-serpentcs) [reply business_requirement_deliverable_default/models/business_requirement_deliverable_default.py]: @elicoidal Sorry, I did that in XML but now I have done that in py as well.
Thanks for drawing the attention.

**Review by astirpe — APPROVED** [review]

**Comment by elicoidal** [comment]
Travis needs some love @sudhir-serpentcs :)

**Comment by sudhir-serpentcs** [comment]
@elicoidal Fixed travis issue.

**Pull request merged**
